### PR TITLE
feat: typography and layout tweaks for /donate/supportus

### PR DESCRIPTION
**Pull request by ryan-ludwig**
https://test.kiva.org/donate/supportus
https://www.kiva.org/donate/supportus

### CODE REVIEW

File: src/pages/Donate/DonateSupportUsRightRail.vue [hunk 1]
```diff
@@ -69,7 +70,7 @@ export default {
 				item => item.key === 'web-donate-support-us-callout-2'
 			) || {};
 			const fundInnovationImage = this.content?.media?.find(
-				image => image.title === 'Globe Grid Illustration'
+				image => image.title === 'homepage-stats-icon-3'
 			);
```
Comment (ryan-ludwig) [inline src/pages/Donate/DonateSupportUsRightRail.vue:74]: This illustration from contentful was a different size than the piggy bank. It had transparent padding within the image itself.
image

The new one is a touch different but lines up correctly 
image

File: src/pages/Donate/DonateForm.vue [hunk 2]
```diff
@@ -25,9 +25,7 @@
 					v-model="isMonthly"
 					v-kv-track-event="['Donate form', 'toggle-monthly-donation', 'Make a monthly donation.']"
 				>
-					<template #after>
-						Make a monthly donation.
-					</template>
+					Make a monthly donation.
```
Comment (mcstover) [inline src/pages/Donate/DonateForm.vue:28]: Guessing this just works and looks right without using the `#after` slot...

**Review by mcstover — APPROVED** [review]

File: src/pages/Donate/DonateForm.vue (hunk 2 above)
Comment (ryan-ludwig) [reply src/pages/Donate/DonateForm.vue:28]: Yeah, part of the earlier change to kv-base-input

File: src/pages/Donate/DonateSupportUsRightRail.vue (hunk 1 above)
Comment (mcstover) [reply src/pages/Donate/DonateSupportUsRightRail.vue:74]: Thank you! I think that image was potentially changed for a different context after it was initially used here...

**Review by roger-in-kiva — APPROVED** [review]
Good job!

**Pull request merged**